content/introduction.html

```diff
@@ -2,13 +2,13 @@ <h1> About the program </h1>
 <p class="full">
   The University of Waterloo’s Software Engineering (SE) is an interdisciplinary
   program supported by both the Faculty of Mathematics and the Faculty of
-  Engineering. For 5 years, all 141 graduating students of the class of 2018 spent 8
+  Engineering. For 5 years, all 92 graduating students of the class of 2021 spent 8
   semesters in school with similar schedules as well as 6 co-ops.
   <br /><br />
   SE is commonly compared to Computer Science, and is often seen as its more-constrained,
   engineering-oriented counterpart. Prospective students may wonder whether SE
   is the right fit for them, and whether they’ll enjoy the environment. This
-  profile attempts to showcase who SE 2018 students are, what they've done, and
+  profile attempts to showcase who SE 2021 students are, what they've done, and
   how university went for students.
```

Comment:
"This profile attempts to showcase who SE 2021 students are, what they've done, and how they had experienced university."
(something something grammar and matching subjects in connected clauses)